grammar.js

```diff
@@ -199,7 +199,13 @@ module.exports = grammar({
             ),
         ),
 
-        interval: $ => {
+        interval: $ => choice(
+            $._interval_period,
+            $._interval_date_spec,
+            seq($._interval_period, $.whitespace, $._interval_date_spec)
```

Comment:
I considered capturing these into named nodes, but opted not to because it I didn't need it for my purpose, and it could be added later if needed. I'm referring to something like `(interval (period) (duration))`, instead of just `(interval)`.

Reply:
No worries, this is open source, not a day job. "Just" scratching your own itches is a lot better than doing nothing!